### ISSUE TITLE
Validation scenes exhibit low PSNR

### ISSUE THREAD

**Issue by JamChaos** [OpenRobotLab/AnySplat#91]
Hi, thank you for your great work. I am currently training AnySplat on ScanNet++ and have run more than 6,000 iterations. I observe that the PSNR on the training set is normal, but the PSNR on the validation set is very poor. What could be the reason for this?

<img width="3690" height="480" alt="Image" src="https://github.com/user-attachments/assets/d8b8234d-5eed-4062-885d-5514ad37ea58" />

<img width="1317" height="516" alt="Image" src="https://github.com/user-attachments/assets/be0932d2-18a5-4122-a78d-cb7bf6d92e57" />

**Comment by JamChaos** [comment]
changed https://github.com/InternRobotics/AnySplat/blob/5f5e208a7dd57d52e43ea0d553a95eab526e8775/src/model/model_wrapper.py#L476 to         encoder_output, output = self.model((batch["context"]["image"] + 1) / 2, self.global_step, visualization_dump=visualization_dump)
